### ISSUE TITLE
Identify and implement ways to improve board functionality [0/?]

### ISSUE THREAD

**Issue by RobertDuras** [Enari/DVA313#128]
This is an issue related to improving the way the board works in terms of backend code. 
It builds off of and attempts to improve upon the logic that is being implemented with issue #113 . 
I’ve already started on implementing parts of this, but I’ll leave those part of the functionality out for now as they aren’t done. 
The reason I’m putting this in out of scope is that the logic for this is getting pretty complex, so going for it directly wouldn’t give the smallest viable product. There’s an alternative easier way that doesn’t require as much work to implement so I’ll go with at first. (Once #55 is done)
I’ll most likely come back to this later if we have time left to do so.

**Comment by Icepool** [comment]
That's a better name for it as this isn't strictly related to only the updating of the board. 
It's backend in general with a focus on improving the update functionality.
